integration_tests/__tests__/__snapshots__/multi_project_runner.test.js.snap

```diff
@@ -10,29 +10,31 @@ Ran all test suites.
 `;
 
 exports[`can pass projects or global config 2`] = `
-"Test Suites: 2 passed, 2 total
-Tests:       2 passed, 2 total
+"Test Suites: 3 passed, 3 total
+Tests:       3 passed, 3 total
 Snapshots:   0 total
 Time:        <<REPLACED>>
-Ran all test suites in 2 projects.
+Ran all test suites in 3 projects.
 "
 `;
 
 exports[`can pass projects or global config 3`] = `
-"PASS  project1/__tests__/file1.test.js
+"PASS   BACKEND  project1/__tests__/file1.test.js
+PASS   UI  project3/__tests__/file1.test.js
```

Comment:
@aaronabramov There is a double space here... this happens in cases where we disable `chalk`... because we are doing
```js
chalk.someOptions(` ${displayName} `)
```

Should we remove those extra spaces when chalk is disabled?

Reply:
Yes, let's remove it in that case. Both on the left and right.

Reply:
I thought that only the `displayName` had the extra spaces but it seems that PASS/FAIL also have them already in master... so the fix might involve a bit more changes
<img width="403" alt="screen shot 2017-08-24 at 11 16 33 pm" src="https://user-images.githubusercontent.com/574806/29701731-62b64bfc-8922-11e7-9962-326734abc6e9.png">

Reply:
Here is the diff for removing the extra spaces in `PASS` and `FAIL` https://github.com/rogeliog/jest/compare/add-display-name...rogeliog:add-display-name-full-diff

@cpojer @aaronabramov I'm not sure if that leading space before PASS and the two after it are intentionally there in the current version or if it is fine to remove them? I'm fine with either I just wanted to understand if any of these current behavior was intentional... I'll update this PR after I get a better understanding on the expected behavior of non interactive non MPR output :) thanks!

Reply:
I think it's fine to remove them when colors are disabled.